src/view/ViewCommandHandlers.js

```diff
@@ -515,6 +533,20 @@ define(function (require, exports, module) {
         setFontFamily(prefs.get("fontFamily"));
     });
 
+    // Define a preference for font smoothing mode on Mac.
+    // By default fontSmoothing is set to "subpixel-antialiased"
+    // for the text inside code editor. It can be overridden
+    // to "antialiased", that would set text rendering AA to use
+    // gray scale antialiasing.
+    if (brackets.platform === "mac") {
+        prefs.definePreference("fontSmoothing", "string", "subpixel_antialiased", {
```

Comment:
My bad. I had changed it at one place but forgot to change it here. I will make the changes here. Thanks!